### PR TITLE
go version update 1.15.5

### PR DESCRIPTION
**Pull request by mumutyal**
Updating go lag from 1.15.2 to 1.15.5

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ branches:
 
 before_install:
   - git config --global url."https://$GHE_USER:$GHE_TOKEN@github.com/".insteadOf "https://github.com/"
+  - git config --global url."ssh://git@github.ibm.com".insteadOf https://github.ibm.com
```
Comment (akgunjal) [inline .travis.yml:26]: Remove the github.ibm.com config

File: block-storage-attacher/Dockerfile.builder [hunk 2]
```diff
@@ -1,6 +1,6 @@
-FROM golang:1.15.2
+FROM golang:1.15.5
 
-ARG GOPROXY=off
+#ARG GOPROXY=off
```
Comment (akgunjal) [inline block-storage-attacher/Dockerfile.builder:3]: why is this removed?

File: block-storage-attacher/Makefile [hunk 3]
```diff
@@ -80,5 +79,5 @@ build-systemutil:
 
 .PHONY: oss
 oss:
-	go get github.ibm.com/alchemy-containers/armada-opensource-lib@v0.4
-	go run github.ibm.com/alchemy-containers/armada-opensource-lib/cmd/makeoss ${OSS_FILES}
+	#go get github.ibm.com/alchemy-containers/armada-opensource-lib@v0.4
```
Comment (akgunjal) [inline block-storage-attacher/Makefile:82]: Dont remove the `oss` task. Please revert this.

File: block-storage-attacher/go.mod [hunk 4]
```diff
@@ -14,8 +14,6 @@ require (
 	github.com/securego/gosec v0.0.0-20200401082031-e946c8c39989 // indirect
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.4.0
-	github.ibm.com/alchemy-containers/armada-opensource-lib v0.4.4 // indirect
-	github.ibm.com/alchemy-containers/go-build-tools v0.1.18 // indirect
```
Comment (akgunjal) [inline block-storage-attacher/go.mod:18]: you also need to have corresponding go.sum changes

**Review by akgunjal — CHANGES_REQUESTED** [review]

File: block-storage-attacher/Dockerfile.builder (hunk 2 above)
Comment (mumutyal) [reply block-storage-attacher/Dockerfile.builder:3]: If we keep it on we are getting build errors

File: block-storage-attacher/Makefile (hunk 3 above)
Comment (mumutyal) [reply block-storage-attacher/Makefile:82]: This is also from github.ibm.com we will land into same issue

File: .travis.yml (hunk 1 above)
Comment (mumutyal) [reply .travis.yml:26]: If we remove this then oss task wil fail we need this

File: block-storage-attacher/Makefile (hunk 3 above)
Comment (mumutyal) [reply block-storage-attacher/Makefile]: reverted

**Review by akgunjal — APPROVED** [review]
LGTM
